content/reference/sql/version-control/merges.md

```diff
@@ -171,3 +194,76 @@ or with the [`@@dolt_transaction_commit` system
 variable](./dolt-sysvars.md#dolt_transaction_commit)) if the working
 set has conflicts. You must resolve conflicts before creating a Dolt
 commit.
+
+# Constraint violations
+
+If any table in your database contains foreign key constraints or unique key
+constraints, it's possible that a merge will create constraint violations. When
+this occurs, the `conflicts` column will be set to `1`.
+
+Let's walk through an example for foreign key constraints. 
+
+## Foreign Key Constraint Violations
+
+Imagine we have a parent table and a child table with this schema:
+```sql
+CREATE TABLE parent (pk INT PRIMARY KEY);
+CREATE TABLE child (
+    pk INT PRIMARY KEY, 
+    parent_fk INT, 
+    FOREIGN KEY (parent_fk) REFERENCES parent (pk)
+);
+```
+
+Let's create a situation where a merge adds a child to a parent that no longer
+exists:
+```sql
+INSERT INTO parent values (1);
+CALL DOLT_COMMIT('-Am', 'setup');
+
+CALL DOLT_CHECKOUT('-b', 'branch_to_merge');
+INSERT INTO child values (1, 1);
+CALL DOLT_COMMIT('-Am', 'add a child of parent 1');
+
+CALL DOLT_CHECKOUT('main');
+DELETE from parent where pk = 1;
+CALL DOLT_COMMIT('-Am', 'delete parent 1');
+```
+
+When we merge, we see the `conflict` column has been set: 
```

Comment:
I'm going to split the `conflicts` column into `conflicts` and `constraint_violations` when some of the new format bugs are gone.